backend/cmmgen.ml

```diff
@@ -1487,8 +1484,14 @@ let transl_function f =
     else
       [ Reduce_code_size ]
   in
+  let params_layout =
+    if List.length f.params = List.length f.arity.params_layout then
+      f.arity.params_layout
+    else
+      f.arity.params_layout @ [Lambda.layout_function]
```

Comment:
Maybe `layout_closure` would be a better name?

Reply:
Will change in a separate PR.